src/Evolution/Executables/ScalarAdvection/EvolveScalarAdvection.hpp

```diff
@@ -33,6 +33,9 @@
 #include "Evolution/Systems/ScalarAdvection/BoundaryConditions/Factory.hpp"
 #include "Evolution/Systems/ScalarAdvection/BoundaryCorrections/Factory.hpp"
 #include "Evolution/Systems/ScalarAdvection/BoundaryCorrections/RegisterDerived.hpp"
+#include "Evolution/Systems/ScalarAdvection/FiniteDifference/Factory.hpp"
```

Comment:
can you make switching the executable to subcell its own commit? The action list is also wrong for subcell, so I'm not sure what this part of the commit is supposed to be doing

Reply:
I'll include this fractional change into a separate commit for switching executable